### PR TITLE
Adding outpost subnet requirement to VPC install doc

### PR DESCRIPTION
**Pull request by bscott-rh**
Version(s):
4.15

Issue:
https://issues.redhat.com/browse/OSDOCS-9286

Link to docs preview:
https://72092--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-vpc#installation-custom-aws-vpc-requirements_installing-aws-vpc (added one bullet point)

QE review:
- [x] QE has approved this change.

Adding extra requirement for Outpost subnet tags in VPC install doc per Marco Braga

### CODE REVIEW

**Review by mtulio — COMMENTED** [review]
/lgtm

/assign @makentenza @yunjiang29 @JoelSpeed

**Review by JoelSpeed — COMMENTED** [review]
/lgtm

**Review by makentenza — APPROVED** [review]
LGTM

**Comment by kalexand-rh** [comment]
The `branch/enterprise-4.16` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.15. And any PR going into `main` must also target the latest version branch (enterprise-4.16).

**Review by kalexand-rh — APPROVED** [review]

**Review by michaelryanpeter — COMMENTED** [review]
Feedback to consider. I tried to clear up sections where I lost the thread a little.

File: modules/installation-custom-aws-vpc.adoc [hunk 1]
```diff
@@ -99,6 +99,7 @@ endif::aws-outposts[]
 * The VPC must not use the `kubernetes.io/cluster/.*: owned`, `Name`, and `openshift.io/cluster` tags.
 +
 The installation program modifies your subnets to add the `kubernetes.io/cluster/.*: shared` tag, so your subnets must have at least one free tag slot available for it. See link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html#tag-restrictions[Tag Restrictions] in the AWS documentation to confirm that the installation program can add a tag to each subnet that you specify. You cannot use a `Name` tag, because it overlaps with the EC2 `Name` field and the installation fails.
+* If you intend to extend your {product-title} cluster into an AWS Outpost, and you have an existing Outpost subnet, that subnet must use the `kubernetes.io/cluster/unmanaged: true` tag. If this tag is not applied, the installation may fail due to the Cloud Controller Manager creating a service load balancer in the Outpost subnet, which is an unsupported configuration.
```
[the remark this answers is not in the document]
Comment (bscott-rh) [reply modules/installation-custom-aws-vpc.adoc:102]: Good feedback, thank you

**Pull request merged**